cypress/integration/attributes/check-cluster-attributes-singleton-in-zigbee-mode.spec.js

```diff
@@ -0,0 +1,115 @@
+// one global cluster specific attribute or command value/setting shared across endpoints.
+// They are not endpoint specific.
+
+describe('Testing endpoints sharing attribute values', () => {
+  it('create Endpoint 1', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.gotoAttributePage('Billing Unit (0x0203)', 'General')
+  })
+  it(
+    'set initial values on Endpoint 1',
+    { retries: { runMode: 2, openMode: 2 } },
+    () => {
+      // Set ZCL version - 1
+      cy.get(
+        ':nth-child(1) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('1', { force: true })
+
+      // Set power source - 22
+      cy.get(
+        ':nth-child(8) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('22', { force: true })
+
+      // Set cluster revision - 33
+      cy.get(
+        ':nth-child(19) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      )
+        .clear({ force: true })
+        .type('33', { force: true })
+
+      // Set ResetToFactoryDefaults command
+      cy.gotoCommandsTab()
+      cy.get(':nth-child(3) > .q-mt-xs > .q-checkbox__inner > input').check({
+        force: true,
+      })
+    }
+  )
+  it('create Endpoint 2', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+
+    cy.gotoAttributePage('Chatting Station (0x0601)', 'General')
+  })
+  it(
+    'verify initial values on Endpoint 2 are reflected from Endpoint 1',
+    { retries: { runMode: 2, openMode: 2 } },
+    () => {
+      // Set ZCL version - 1
+      cy.get(
+        ':nth-child(1) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      ).should('have.value', '1')
+
+      // Set power source - 22
+      cy.get(
+        ':nth-child(8) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      ).should('have.value', '22')
+
+      // Set cluster revision - 33
+      cy.get(
+        ':nth-child(19) > [style="min-width: 180px;"] > .q-field > .q-field__inner .q-field__control > .q-field__control-container > input'
+      ).should('have.value', '33')
+
+      // Set ResetToFactoryDefaults command
+      cy.gotoCommandsTab()
+      cy.get(':nth-child(3) > .q-mt-xs').should(
+        'have.attr',
+        'aria-checked',
+        'true'
+      )
+
+      // Uncheck ResetToFactoryDefaults command
+      cy.get(':nth-child(3) > .q-mt-xs')
+        .click({
+          force: true,
+        })
+        .should('have.attr', 'aria-checked', 'false')
+    }
+  )
+
+  it(
+    'Check Command changes on Endpoint 2 is reflected on Endpoint 1',
```

Comment:
How are you doing this? Not very familiar with cypress tests so looking at the code below I am confused.

Reply:
So the `cy.gotoAttributePage` and `cy.gotoCommandsTab()` take you to Ep1? Could we name them in a way that mentions EP1 as well for less confusion.